src/Installer/ReleaseNotes.md

```diff
@@ -1,4 +1,12 @@
 ## DEV_VERSION_NUMBER: DEV_RELEASE_DATE
+* Chorus: Update documentation of Chorus Hub
+* FLExBridge: Improve developer documentation and update documentation of recent fixes
```

Comment:
Yes, users will see this file. Sparkle will show it to users before they install it, so they can decide if they want what the new version contains.